### ISSUE TITLE
Multiple source directories?

### ISSUE THREAD

**Issue by DeviantEng** [saimn/sigal#225]
I'm relatively new to Sigal, but am really happy with it.  Is it possible to specify multiple source directories?  For example, I have it currently searching `/mnt/dhorn_photos/Phone` so that it finds photos that get synced to my server from my phone (each phone model has a folder inside), but I'd also like to include `/mnt/dhorn_photos/Camera` to also get photos taken with my GoPro and Phantom 4.  

I have other folders inside `/mnt/dhorn_photos` I don't want Sigal touching (such as my Lightroom dir, some scanned images, etc), and I wouldn't want Sigal to create sub-dirs if I was to use `/mnt/dhorn_photos` as my root.  

So, can I specify multiple sources?  Or can I specify dirs that Sigal should ignore?

Thanks!

**Comment by DeviantEng** [comment]
Just found #63 for ignoring dirs, so awesome!

Still, is it possible to select multiple source directories?

**Comment by saimn** [comment]
Hi @DeviantEng 
You can indeed specify a list of directories to ignore.
About the multiple source directories, it is not supported currently. The issue is that we need a root directory that will be the home page of the gallery, and all links are relative to this root directory. So it should be possible to handle this, but I'm not sure how complex it is in terms of code.
Meanwhile, a workaround is to use symbolic links. You could create a directory for Sigal and link the directories you want to include in it.

**Comment by DeviantEng** [comment]
@saimn Thanks for the reply and the possible options.  Luckily, I was able to do what I wanted to do using a list of directories to ignore.  Logically, it took me a bit to figure out how to make it work for me, but I did.  

I understand the complexity with adding multiple directories, and from a coding perspective I can't really be of any help.  Feel free to close this issue if you'd rather not worry about adding that ability at this time.

Thanks.
